android-components/components/feature/search/src/main/assets/search/list.json

```diff
@@ -822,27 +822,27 @@
     "tr": {
       "default": {
         "visibleDefaultEngines": [
-          "google-b-m", "ddg", "wikipedia-tr"
+          "google-b-m", "bing", "ddg", "wikipedia-tr"
         ]
       },
       "TR": {
         "visibleDefaultEngines": [
-          "google-com-nocodes", "ddg", "wikipedia-tr"
+          "google-com-nocodes", "ddg", "bing", "wikipedia-tr"
         ]
       },
       "BY": {
         "visibleDefaultEngines": [
-          "google-com-nocodes", "ddg", "wikipedia-tr"
+          "google-com-nocodes", "ddg", "bing", "wikipedia-tr"
         ]
       },
       "KZ": {
         "visibleDefaultEngines": [
-          "google-com-nocodes", "ddg", "wikipedia-tr"
+          "google-com-nocodes", "ddg", "bing", "wikipedia-tr"
         ]
       },
       "RU": {
         "visibleDefaultEngines": [
-          "google-com-nocodes", "ddg", "wikipedia-tr"
+          "google-com-nocodes", "ddg", "bing", "wikipedia-tr"
```

Comment:
Same as above, but there wasn't any noted missing search engine in the spreadsheet even though I can visually confirm that bing is missing.